core/src/main/scala/org/genivi/sota/core/db/AutoInstallPackages.scala

```diff
@@ -47,7 +48,7 @@ object AutoInstalls {
 
   def addDevice(ns: Namespace, pkgName: PackageId.Name, dev: Uuid)
                (implicit ec: ExecutionContext): DBIO[Unit]
-    = all.insertOrUpdate(AutoInstall(ns, pkgName, dev)).map(_ => ())
```

Comment:
what happened to insertOrUpdate?

This is not the same though. This will fail if auto install already exists.